apps/web/cypress/tests/auth.spec.ts

```diff
@@ -130,12 +189,15 @@ describe('User Sign-up and Login', function () {
       cy.getByTestId('password').type('123qwe!@#');
       cy.getByTestId('submit-btn').click();
 
+      cy.location('pathname').should('equal', '/templates');
+
       // setting current time in future, to simulate expired token
-      const todaysDate = new Date();
-      todaysDate.setDate(todaysDate.getDate() + 30); // iat - exp = 30 days
-      cy.clock(todaysDate);
+      const THIRTY_DAYS = 1000 * 60 * 60 * 24 * 30; // iat - exp = 30 days
+      const ONE_MINUTE = 1000 * 60; // adding 1 minute to be sure that token is expired
+      const date = new Date(Date.now() + THIRTY_DAYS + ONE_MINUTE);
+      cy.clock(date);
 
-      cy.visit('/templates');
+      cy.visit('/subscribers');
```

Comment:
I had to update this test as I have had issues that the `cy.clock(date)` was before the token exp date, even if the token was generated before, weird, so I've added one additional minute to that date.